tests/test_qnode_legacy.py

```diff
@@ -404,6 +404,45 @@ def test_not_giving_mode_kwarg_does_not_raise_warning(self):
         assert len(record) == 0
 
 
+# pylint: disable=too-few-public-methods
```

Comment:
Probably don't need to add tests here anymore, since we no longer have duplicate logical paths for the two different device interfaces.